frontend/util/export-to-csv.component.tsx

```diff
@@ -0,0 +1,57 @@
+// from https://codeburst.io/export-objects-array-as-csv-using-typescript-643bf4f794d9
+
+export class CsvDataService {
```

Comment:
Using classes for functions like this in javascript is unnecessary and unconventional in the react community. Better is just to export the functions directly:

```js
export function exportToCsv(filename, rows) {
```